db/sample_data/events.rb

```diff
@@ -74,7 +74,7 @@ def event_gongakrobatik
       end_date: Date.new(2021, 03, 05)
   )
   Event.new(
-      name: 'Einführung in die Kunst der Gongakrobatik',
+      name: 'Gongakrobatik',
```

Comment:
Just because we do not want to have so huge names in order to keep the UI simple.